paper/abstract.md

```diff
@@ -0,0 +1,20 @@
+
+
+As higher education decision makers increasingly turn to statistical
+and machine learning algorithms to predict student success, they often
+do so without a clear understanding of how these algorithms work or
+the tradeoffs made when basing decisions upon their predictions. Of
+particular concern are hidden biases inherent in an uncritical use of
+these algorithms that negatively affect marginalized student
+populations. In this project we use `R` to transparently build a
+number of predictive algorithms and show how decisions based on their
+results might affect underserved student groups in higher
+education. We first use publicly available data from the High School
+Longitudinal Study and the `tidymodels` framework to evaluate the
+predictive accuracy of different modeling approaches, including
+elastic net regression, random forests, and neural networks. We next
+use predictions from these models to simulate the impact of various
+decision frameworks on students, with a particular focus on how
```

Comment:
I don't know what is meant by "decision frameworks"

sorry for tardiness. cuz I don't know what it means to review a pull request

Reply:
Thanks, everyone!  @ozanj you seemed to get it, but just in case, the idea of reviewing a pull request to let someone else take a look at the work that's been done prior to merging it into the main project. I generally wouldn't assign everyone to review a pull request, but I did this time in no small part because it gave us a chance to agree as to what we're working on.  Let me know if I can clarify further.